gopt/gopt.h

```diff
@@ -34,6 +34,12 @@ struct option
   /* input/output: */
   unsigned int  flags;
 
+  /* char* array allocated by caller, each char* filled by gopt. GOPT_REPEATABLE_VALUE only. */
+  char        **arguments;
+
+  /* input, GOPT_REPEATABLE_VALUE only. */
```

Comment:
Not just only used when GOPT_REPEATABLE_VALUE is set, but required if GOPT_REPEATABLE_VALUE is set.